### ISSUE TITLE
NVDA reads layout tables in Outlook 365

### ISSUE THREAD

**Issue by sarahjoga** [nvaccess/nvda#14892]
### Steps to reproduce:
When in Outlook 365, enter the message view (F6) and then read the email (Insert downarrow)

### Actual behavior:
When the readout gets to the email content, the layout tables used for email marketing are read, despite role="presentation" being applied to all tables. This happens in all email marketing messages I have in my inbox using NVDA and Outlook 365. The source code pulled from Outlook 365 still includes the role="presentation" - it appears that NVDA is ignoring that and reading the table as a data table. This does not happen on Outlook 2021 or with other screen readers when reading the same emails. Please note that html email marketing cannot work without layout tables.

### Expected behavior:
NVDA should be able to use the role="presentation" attribute to read the tables as layout tables and therefore read the email content.

#### Name and version of other software in use when reproducing the issue:
Microsoft® Outlook® for Microsoft 365 MSO (Version 2303 Build 16.0.16227.20202) 64-bit

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu? I do not know how to do this.

**Comment by Adriani90** [comment]
@sarahjoga is this occuring with UIA enabled? Or disabled? If you check in NVDA advanced settings, the setting "using UIA for Microsoft Word controls", is it set to "only when necessary?" Or is it set to "default (when suitable)"?

Please try with UIA set on "only when necessary". Is the issue still occuring?

Also we need a minimal test case to be able to test this and investigate further. Can you provide a test case please?

cc: @michaelDCurran

**Comment by sarahjoga** [comment]
I had the standard defaults from when I initially installed NVDA, so the Registration for UI Automation is set as "Automatic (prefer selective)" and the UI Automation to access Microsoft Word document controls is set to "Default (where suitable)"

When I switched the Microsoft Word controls to "only when necessary" I had the same results - with NVDA reading out the tables in Outlook 365.

After you suggested this I did mess around a bit and toggled the elements to their opposites to see what would happen. It was not an extensive test, but when I had the UI Automation registration at "Global"; the Microsoft Word doc controls at "Always" the table-reading stopped and it was read as expected. Obviously I did not test anything else - happy to do more testing if you have any direction.

Can you define what you mean by minimal test case? Do you want an HTML file, or a .msg file?

**Comment by Adriani90** [comment]
Ideally a msg file showing such tables and a html file showing the coresponding html code.Von meinem iPhone gesendetAm 01.05.2023 um 23:01 schrieb sarahjoga ***@***.***>:﻿
I had the standard defaults from when I initially installed NVDA, so the Registration for UI Automation is set as "Automatic (prefer selective)" and the UI Automation to access Microsoft Word document controls is set to "Default (where suitable)"
When I switched the Microsoft Word controls to "only when necessary" I had the same results - with NVDA reading out the tables in Outlook 365.
After you suggested this I did mess around a bit and toggled the elements to their opposites to see what would happen. It was not an extensive test, but when I had the UI Automation registration at "Global"; the Microsoft Word doc controls at "Always" the table-reading stopped and it was read as expected. Obviously I did not test anything else - happy to do more testing if you have any direction.
Can you define what you mean by minimal test case? Do you want an HTML file, or a .msg file?

**Comment by sarahjoga** [comment]
[msg_html.zip](https://github.com/nvaccess/nvda/files/11368711/msg_html.zip)

Attached are three files - 1 html file, 1 msg file of that html file after being sent, and a second msg file of a Target email. Each of those msg files produces the table readout like I outlined earlier.
